tests/data/test_VideoDataset.py

```diff
@@ -51,37 +50,29 @@ def create_dataset_specified_frames_per_video(
         self.input_dir = tempfile.mkdtemp()
         self.ensure_dir(self.input_dir)
         self.frames_over_videos = [
-            (np.random.randn(frames, w, h, c) * 255).astype(np.uint8)
+            (torch.randn(frames, w, h, c) * 255).to(torch.uint8)
             for frames in frames_per_video
         ]
         self.extensions = ".avi"
 
         for frames in self.frames_over_videos:
             path = os.path.join(self.input_dir, f"output-{len(frames):03}.avi")
             print(path)
-            out = cv2.VideoWriter(path, 0, 1, (w, h))
-            for frame in frames:
-                out.write(frame)
-            out.release()
+            out = torchvision.io.write_video(path, self.frames_over_videos, frames)
```

Comment:
```suggestion
            out = torchvision.io.write_video(filename=path, video_array=frames, fps=1)
```

`self.frames_over_videos` is a list of videos and we want to save every video individually.